library/psa_crypto.c

```
@@ -7916,6 +7918,14 @@ static psa_status_t psa_validate_key_type_and_size_for_key_generation(
     } else
 #endif /* defined(PSA_WANT_KEY_TYPE_ECC_KEY_PAIR_GENERATE) */
 
+#if defined(PSA_WANT_KEY_TYPE_MLKEM_KEY_PAIR_GENERATE)
+    if (PSA_KEY_TYPE_IS_MLKEM(type) && PSA_KEY_TYPE_IS_KEY_PAIR(type)) {
+        /* To avoid empty block, return successfully here. */
+        // REVISIT: KF do we want logic here?
+        return PSA_SUCCESS;
```

I would return a failure here till we merge in an mbedtls implementation assuming they use this path. 
Probably update the comment block to indicate as much for future reference.

---

This path is used, ECC right above does the same thing of returning success without doing any checks on the size.

But some of the other algorithms do have size checks. So the revisit was more if we wanted to implement a size check. In any case, we need this function to succeed